### PR TITLE
Arc 2274 analytics for spa  add events

### PR DESCRIPTION
**Pull request by gxueatlassian**
**What's in this PR?**
1. Send analytics events for various buttons
2. Refactor some of the routes

**Why**
1. Send analytics

**Added feature flags**

**Affected issues**  
ARC-2274

**How has this been tested?**  

**Whats Next?**

### CODE REVIEW

File: etc/poco/bundle/main-test.json [hunk 1]
```diff
@@ -96,14 +96,6 @@
 			"principals": [],
 			"allowed": true
 		},
-		{
```
Comment (gxueatlassian) [inline etc/poco/bundle/main-test.json:99]: I've change to use the `/public/assets` to load those svgs, as those are assets from public folder, it make more sense to name it like that.

File: spa/src/analytics/context.ts [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+	tenantId: string;
+	accountId: string;
+	clientKey: string;
+};
+
+export const getUserContext = async (): Promise<UserContext | undefined> => {
+
+	try {
+
+		const jwtToken: string = await new Promise((res) => {
```
Comment (krazziekay) [inline spa/src/analytics/context.ts:13]: Can we reuse the `getHeaders` from `axiosInstance`? Maybe give it a different name like `getAtlassianJWT`.

File: spa/src/analytics/sox-analytics-client.ts [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+				}
+			);
+
+			getUserContext()
+				.then((userContext: UserContext | undefined) => {
+					if (userContext) {
+						client.setTenantInfo(imported.tenantType.CLOUD_ID, userContext.tenantId);
+						client.setUserInfo(imported.userType.ATLASSIAN_ACCOUNT, userContext.accountId);
+						client.setUIViewedAttributes({ clientKey: userContext.clientKey });
+						console.info("analytis loaded");
```
Comment (krazziekay) [inline spa/src/analytics/sox-analytics-client.ts:26]: /nit
```suggestion
						console.info("analytics loaded");
```

File: src/routes/spa/spa-router.ts [hunk 4]
```diff
@@ -0,0 +1,19 @@
+import { Router, static as Static } from "express";
+import path from "path";
+import fs from "fs";
+import { envVars } from "config/env";
+
+export const SpaRouter = Router();
+
+const rootPath = process.cwd();
+
+//Asset from within the new spa expereicen in /spa/build/static
```
Comment (krazziekay) [inline src/routes/spa/spa-router.ts:10]: \nit
```suggestion
//Assets from within the new spa experience in /spa/build/static
```

**Review by krazziekay — APPROVED** [review]
Small nits, not much of an issue though.

File: spa/src/analytics/context.ts (hunk 2 above)
Comment (gxueatlassian) [reply spa/src/analytics/context.ts]: done.

**Review by krazziekay — DISMISSED** [review]

**Review by krazziekay — APPROVED** [review]

**Pull request merged**